CMake/fletch-tarballs.cmake

```diff
@@ -261,8 +261,8 @@ list(APPEND fletch_external_sources libkml)
 # Support 4.8.6 and 5.11 optionally
 if (fletch_ENABLE_Qt OR fletch_ENABLE_VTK OR fletch_ENABLE_qtExtensions OR
     fletch_ENABLE_ALL_PACKAGES)
-  set(Qt_SELECT_VERSION 5.11.2 CACHE STRING "Select the version of Qt to build.")
-  set_property(CACHE Qt_SELECT_VERSION PROPERTY STRINGS "4.8.6" "5.11.2")
+  set(Qt_SELECT_VERSION 5.12.2 CACHE STRING "Select the version of Qt to build.")
+  set_property(CACHE Qt_SELECT_VERSION PROPERTY STRINGS "4.8.6" "5.12.2")
 
   set(Qt_version ${Qt_SELECT_VERSION})
```

Comment:
BTW, I wonder why we do this rather than have the user selected version be only X.Y (like VTK) and set the full version in the `if` chain just below? (This would also allow replacing a hard-coded version in the URL with `${Qt_SELECT_VERSION}`... in fact, I think the URL would not even need to be conditional?)

Reply:
Some packages do it the right way and some not. I think it would be worthwhile to go through them all and cleanup as a separate branch. I would rather not do that here so we can get this one landed asap.